Dockerfile

```diff
@@ -16,12 +16,12 @@ RUN set -x && \
     # Clean-up
     apk del .deps
 
+ARG DOCKER_COMPOSE_VERSION
+ENV DOCKER_COMPOSE_VERSION=${DOCKER_COMPOSE_VERSION:-"1.10.0"}
```

Comment:
If you want automatically the latest version you need a shell script to determine that because GitHub has not a direct link for that.